Tests/BlockSetTests/Public.swift

```diff
@@ -206,7 +206,7 @@ func model(_ cas: Cas) throws {
         let x = try cas.listMutable()
         #expect(x.count == 1)
         let r: Model<ReceiptModel> = try cas.loadJsonModel(x[0])!
-        #expect(r.value.price == "$1.00")
+        #expect(r.value.price == 1.00)
```

Comment:
We should not use `Double` (or other floating point types) for money. Also, the previous version (as a string) had the currency sign `$`. In theory, one document may contain multiple currencies.